### DISCUSSION TITLE
{"error": "This domain is only used to serve REST API"}

### DISCUSSION THREAD

**Discussion by Can1425** [walinejs/waline#1696] [Q&A] [answered]
{"error": "This domain is only used to serve REST API"}

![Screenshot_20221225_114710_alook browser](https://user-images.githubusercontent.com/101759631/209456515-5417651d-20a3-4a9c-8839-4b0782f791f3.jpg)
怎么办，大佬们

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/guide/get-started/ 快速上手多看一下，客户端需要填得是 Vercel 的地址，不是 LeanCloud。
